<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolbxMain</class>
 <widget class="QMainWindow" name="VolbxMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volbx</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="margin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImportData"/>
    <addaction name="actionFilters"/>
    <addaction name="actionHistogram"/>
    <addaction name="actionExport"/>
    <addaction name="actionSaveDatasetAs"/>
    <addaction name="separator"/>
    <addaction name="actionLogs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTemp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTutorials"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTemp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Tool bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImportData"/>
   <addaction name="actionFilters"/>
   <addaction name="actionExport"/>
   <addaction name="actionSaveDatasetAs"/>
   <addaction name="separator"/>
   <addaction name="actionBasic_plot"/>
   <addaction name="actionGroup_plot"/>
   <addaction name="actionHistogram"/>
   <addaction name="separator"/>
   <addaction name="actionLogs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionFilters">
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Icons/Images/filter.png</normaloff>:/Icons/Images/filter.png</iconset>
   </property>
   <property name="text">
    <string>Filters</string>
   </property>
  </action>
  <action name="actionLogs">
   <property name="text">
    <string>Logs</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Icons/Images/charts.png</normaloff>:/Icons/Images/charts.png</iconset>
   </property>
   <property name="text">
    <string>Histogram plot</string>
   </property>
   <property name="toolTip">
    <string>Histogram plot</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Icons/Images/export.png</normaloff>:/Icons/Images/export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionSaveDatasetAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save data</string>
   </property>
  </action>
  <action name="actionSendErrorOrIdea">
   <property name="text">
    <string>Submit bug or idea</string>
   </property>
  </action>
  <action name="actionBugReport">
   <property name="text">
    <string>Check submission status</string>
   </property>
  </action>
  <action name="actionImportData">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionTutorials">
   <property name="text">
    <string>Tutorials</string>
   </property>
  </action>
  <action name="actionCheckForNewVersion">
   <property name="text">
    <string>Check for updates now</string>
   </property>
  </action>
  <action name="actionUpdateAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update automatically</string>
   </property>
   <property name="toolTip">
    <string>Check for updates on start</string>
   </property>
  </action>
  <action name="actionBasic_plot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Icons/Images/basics.png</normaloff>:/Icons/Images/basics.png</iconset>
   </property>
   <property name="text">
    <string>Basic plot</string>
   </property>
   <property name="toolTip">
    <string>Basic plot</string>
   </property>
  </action>
  <action name="actionGroup_plot">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Resources/Resources.qrc">
     <normaloff>:/Icons/Images/grouping.png</normaloff>:/Icons/Images/grouping.png</iconset>
   </property>
   <property name="text">
    <string>Group plot</string>
   </property>
   <property name="toolTip">
    <string>Group plot</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
